scripts/live_test/generate_index.py

```diff
@@ -148,17 +163,19 @@ def render(data, container, container_url, testdata, USER_REPO, USER_BRANCH, COM
       </tr>
     """.format(testdata.total[1], testdata.total[2], testdata.total[3])
 
-    for module, passed, failed, rate in testdata.modules:
+    sorted_modules = sorted(testdata.modules, key=sort_by_module_name)
+
+    for module, passed, failed, rate in sorted_modules:
         reports = ''
         for x in data:
             name = x['name']
             url = x['url']
             if name.startswith(module + '.'):
                 display_name = 'report'
-                if 'parallel' in name:
-                    display_name = 'parallel'
-                elif 'sequential' in name:
-                    display_name = 'sequential'
+                # if 'parallel' in name:
+                #     display_name = 'parallel'
+                # elif 'sequential' in name:
+                #     display_name = 'sequential'
```

Comment:
Any reason to left this comment here?

Reply:
In the past, each module will generate two reports, one is parallel and the other is sequential. If it is multi-platform, there will be more reports. Now I have integrated them into one report, so these codes are not needed.
Beofre:
![image](https://github.com/Azure/azure-cli/assets/18628534/f85db689-2784-46bd-97ef-5c6047a6a0b4)
After:
![image](https://github.com/Azure/azure-cli/assets/18628534/f45937d5-2b10-4f4d-a749-217d00db7b79)